apps/studio/src/components/AppNavbar/AppNavbar.tsx

```diff
@@ -62,6 +99,12 @@ export function AppNavbar(): JSX.Element {
           >
             Report an issue
           </Button>
+          {publishButtonProps.showPublish && (
+            <PublishButton
+              pageId={publishButtonProps.pageId}
+              siteId={publishButtonProps.siteId}
+            />
+          )}
```

Comment:
question(non-blocking): should the button be completely hidden, or should it be shown but hidden?

Reply:
Completely hiding might confuse the user as to why they can't publish. I prefer to show but disable and add a Tooltip saying:
"Publish will be enabled if there are unpublished edits"

@sehyunidaaa lmk your thoughts!